SU2_CFD/src/solver_structure.cpp

```diff
@@ -1,4 +1,4 @@
-/*!
+﻿/*!
  * \file solver_structure.cpp
  * \brief Main subrotuines for solving direct, adjoint and linearized problems.
```

Comment:
Could you please fix that old typo for ‘subroutines’ in the \brief? Also, we don’t solve any linearized gov eqns any longer, so perhaps adjust to ‘primal and adjoint problems’

Reply:
Done